pc-app/src/js/login.js

```diff
@@ -0,0 +1,128 @@
+  var userList = document.querySelector(".user-list");
+  
+  var userDiv = document.createElement("div");
+  userDiv.classList.add("user");
+
+  var usernameSpan = document.createElement("span");
+  usernameSpan.classList.add("username");
+  usernameSpan.textContent = username;
+
+  var userButtonsDiv = document.createElement("div");
+  userButtonsDiv.classList.add("user-buttons");
+
+  var loginButton = document.createElement("button");
+  loginButton.classList.add("login");
+  loginButton.textContent = "Login";
+  loginButton.setAttribute("onclick", `goToLogin('${username}')`);
+
+  var resetButton = document.createElement("button");
+  resetButton.classList.add("reset");
+  resetButton.textContent = "Reset";
+  resetButton.setAttribute("onclick", `goToReset('${username}')`);
+
+  userDiv.appendChild(usernameSpan);
+  userDiv.appendChild(userButtonsDiv);
+
+  userButtonsDiv.appendChild(loginButton);
+  userButtonsDiv.appendChild(resetButton);
+
+  userList.appendChild(userDiv);
+}
+
+function handleLoginSubmit(event) { // eslint-disable-line no-unused-vars
+  event.preventDefault();
+  var password = document.getElementById("password").value;
+  ipcRenderer.send('login:submit', {username, password});
+}
+
+ipcRenderer.on('login:success', (_options) => {
+  goToInterface();
+  console.log("success");
+});
+
+ipcRenderer.on('login:failure', (options) => {
+  console.log("failure: ", options);
+});
+
+function handleResetSubmit(event) { // eslint-disable-line no-unused-vars
+  event.preventDefault();
+  var jwt = document.getElementById("jwt").value;
+  var password = document.getElementById("password").value;
+  ipcRenderer.send('reset:submit', {username, jwt, password});
+}
+
+ipcRenderer.on('reset:success', (_options) => {
+  goToUserlist();
+  console.log("success");
+});
+
+ipcRenderer.on('reset:failure', (options) => {
+  console.log("failure: ", options);
+});
+
+function handleRegisterSubmit(event) { // eslint-disable-line no-unused-vars
+  event.preventDefault();
+  var jwt = document.getElementById("jwt").value;
+  var password = document.getElementById("password").value;
+  ipcRenderer.send('register:submit', {jwt, password});
+}
+
+ipcRenderer.on('register:success', (_options) => {
+  goToUserlist();
+  console.log("success");
+});
+
+ipcRenderer.on('register:failure', (options) => {
+  console.log("failure: ", options);
+});
+
+function goToUserlist() { // eslint-disable-line no-unused-vars
+  window.location.href = 'userlist.html';
+}
+
+function goToLogin(username) { // eslint-disable-line no-unused-vars
+  window.location.href = `login.html?username=${username}`;
+}
+
+function goToReset(username) { // eslint-disable-line no-unused-vars
+  window.location.href = `reset.html?username=${username}`;
+}
+
+function goToRegister() { // eslint-disable-line no-unused-vars
+  window.location.href = `register.html`;
+}
+
+function goToInterface() { // eslint-disable-line no-unused-vars
+  window.location.href = `interface.html`;
```

Comment:
Instead of using iframe, can we use [`loadUrl`](https://www.electronjs.org/docs/latest/api/browser-window#winloadurlurl-options) instead?

Reply:
Sure, I will consider that. Right now that is just for mock testing purposes. Reference to the comment I've left to Jirawut earlier

Reply:
Ok!